tests/filters_test.py

```diff
@@ -0,0 +1,16 @@
+import queue
+
+from tests.isolated_testcase import IsolatedTestCase
+
+
+class FiltersTest(IsolatedTestCase):
+
+    def test_filter_users(self):
+        self.bot_config.RESP_ONLY_REQ_USERS = False
```

Comment:
@nvzard Am I doing anything wrong here?

Reply:
@Vamshi99 `IsolatedTestCase` is loading all the plugins from `plugins/` directory, hence this filter is not loaded by the bot since it is in `utils/` dir.